.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on: 
+  pull_request:
+    branches:
+      - master
+  push:
+    branches-ignore:
+      - '**'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
Hmm, we don't deploy in those jobs. Running tests on the tags is too late. I think they can be removed.

```suggestion
```